mobile_config/src/gateway_service.rs

```diff
@@ -156,16 +156,21 @@ impl mobile_config::Gateway for GatewayService {
         let signer = verify_public_key(&request.signer)?;
         self.verify_request_signature(&signer, &request)?;
 
-        tracing::debug!("fetching all gateways' info");
+        tracing::debug!(
+            "fetching all gateways' info. Device types: {:?} ",
+            request.device_types
```

Comment:
should this debug statement come after the map below? it would debug to a vec of i32s before that conversion, no?

Reply:
Good catch! Fixed